RELEASE_NOTES.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## Changes in Transact 0.2.1:
+
+### Highlights:
+
+* Enable loading .scar files into native `SmartContractArchive`s with the
```

Comment:
Doc guideline: Don't use a hyphen to form a plural.  Instead, try to rewrite to avoid the need for a plural.  Could you say this?

> Enable loading .scar files into a native `SmartContractArchive` with the ...

Reply:
Updated the wording